File: Java/UberBlack.java
import java.util.ArrayList;
import java.util.Map;

class UberBlack extends Car {

    Map<String, Map<String, Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial;
    String brand;
    String model;

    public UberBlack(String license, Account driver, String brand, String model,
            Map<String, Map<String, Integer>> typeCarAccepted, ArrayList<String> seatsMaterial) {

        super(license, driver);
        this.brand = brand;
        this.model = model;
        this.typeCarAccepted = typeCarAccepted;

    }
}

File: Python/UberBlack.py
from Car import Car


class UberBlack():

    typeCarAccepted = []
    seatsMaterial = []

    def __init__(self, license, driver, typeCarAccepted, seatsMaterial) -> None:

        super.__init__(license, driver)
        self.typeCarAccepted = typeCarAccepted
        self.seatsMaterial = seatsMaterial
